services/core-api/app/api/email_tracking/email_status_tasks.py

```diff
@@ -0,0 +1,113 @@
+from datetime import datetime, timezone
+from flask import current_app
+from celery import Task
+
+from app.api.email_tracking.models.email_tracking import EmailTracking, EmailStatus
+from app.api.services.email_service import EmailService
+from app.tasks.celery import celery
+
+
+class EmailTaskBase(Task):
```

Comment:
I'm wondering if we could move this into a shared component, seeing as we've got the exact same logic for the `PermitConditionTaskBase`